python/python.cmd

```diff
@@ -71,15 +71,15 @@ exit /b 1
 REM Make sure there a .hash file that serves as the marker for the source python package the venv is from
 SET PYTHON_VENV_HASH=%PYTHON_VENV%\.hash
 
-IF EXIST %PYTHON_VENV_HASH% GOTO PYTHON_VENV_HASH_EXISTS
+IF EXIST "%PYTHON_VENV_HASH%" GOTO PYTHON_VENV_HASH_EXISTS
 ECHO Python has not been setup completely for O3DE. Missing venv hash %PYTHON_VENV_HASH%
 ECHO Try running %CMD_DIR%\get_python.bat to setup Python for O3DE.
 exit /b 1
 
 :PYTHON_VENV_HASH_EXISTS
 
 REM Read in the .hash from the venv to see if we need to update the version of python
-SET /p VENV_PACKAGE_HASH=<%PYTHON_VENV_HASH%
+SET /p VENV_PACKAGE_HASH=<"%PYTHON_VENV_HASH%"
```

Comment:
Nicely spotted here - this seems strange but I think its correct...

Explanation for other reviewers:
 Normally, `SET` is literal and does not require quotes, eg, 
 ```
 SET somevar=Hello world
 ```
 would set somevar to the string "hello world" since the right hand side of set after the equals is all piped into the variable being set.
 
 so its a bit weird to see quotes in set.... HOWEVER, in this case, `/p` is being used to **prompt from stdin**, and `<` is being used to feed the *filename* stored in the variable `PYTHON_VENV_HASH` in as the stdin contents.  So in this case, the variable is actually a parameter to the `<` pipe-into-stdin operator, not the `set` function and thus must be quoted.